DeviceConnectionAndReliabilityReadme.md

```diff
@@ -0,0 +1,62 @@
+# Azure IoT Device Client C# SDK
+
+## Device connection and messaging reliability
+
+### Overview
+
+In this document you will find information about
+
+- The connection authentication and renewal methods.
+- The reconnection logic and retry policies.
+- The timeout controls.
+
+### Connection authentication
+
+Authentication can be done using either
+
+- [SAS tokens](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-security#security-tokens) - [Sample usage](https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/master/iot-hub/Samples/device/TwinSample/Program.cs#L37)
```

Comment:
would rather link to the reconnection sample here instead, since that is the one we would like to encourage customers to look at

Reply:
Remove `en-us` path